react/src/components/CardsView.js

```diff
@@ -25,7 +25,7 @@ class CardsView extends Component {
   _getUpdatedCards() {
     var self = this;
 
-    $.get(`/cards`).success(function(data) {
+    $.get(`/api/v1/cards`).success(function(data) {
```

Comment:
Note to self: these should be refactored into actions, instead of making requests directly from components